_includes/scripts.html

```diff
@@ -22,18 +22,3 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | replace: _async_marker, '' | relative_url }}"{% if _script.async or _src contains _async_marker %} async{% endif %}></script>
 {% endfor %}{% endfor %}
-
-
-{% if site.search %}
```

Comment:
Was this not working just because of the mismatch in config values, or because of other reasons?

Reply:
This conditional was checking for a `search` key in the site config, which wasn't documented.

Reply:
Right, I'm wondering if the autocomplete would start working if the configuration was fixed, instead of just removing it. I'd rather leave it in if it would start working with the correct config and conditionals in place.

Reply:
It didn't seem to, no. I can try again though.

Reply:
@afeld did you have a chance to try the autocomplete again?

Reply:
Whoooops, I missed this part of their documentation:

> A given term/phrase must be searched 10 times in a 24 hour period to appear as a suggestion.

Confirmed it works using a search.gov affiliate with more traffic ([`usagov`](https://search.usa.gov/search?affiliate=usagov)). Re-added!